lib/spack/spack/fetch_strategy.py

```diff
@@ -712,6 +711,10 @@ def fetch(self):
     def archive(self, destination):
         super(GitFetchStrategy, self).archive(destination, exclude='.git')
 
+    def get_commit_hash(self):
```

Comment:
```suggestion
    def get_commit(self):
```
No need to be specific here.